scripts/teamshow

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+
+"""
+    Script to show LAG and LAG member status in a summary view
+    Example of the output:
+    acsadmin@CO4SCH03010AALF:~$ teamshow
+    Flags: A - active, I - inactive, N/A - Not Available, S - selected, D - deselected
+     No.  Team Dev       Protocol    Ports
+    -----  -------------  ----------  ---------------------------
+       24  PortChannel24  LACP(A)     Ethernet28(S) Ethernet24(S)
+       48  PortChannel48  LACP(A)     Ethernet52(S) Ethernet48(S)
+       16  PortChannel16  LACP(A)     Ethernet20(S) Ethernet16(S)
+       32  PortChannel32  LACP(A)     Ethernet32(S) Ethernet36(S)
+       56  PortChannel56  LACP(A)     Ethernet60(S) Ethernet56(S)
+       40  PortChannel40  LACP(A)     Ethernet44(S) Ethernet40(S)
+        0  PortChannel0   LACP(A)     Ethernet0(S) Ethernet4(S)
+        8  PortChannel8   LACP(A)     Ethernet8(S) Ethernet12(S)
+"""
+
+import json
+import re
+import subprocess
+import sys
+from tabulate import tabulate
+from minigraph import parse_xml
+
+class Teamshow(object):
+    def __init__(self):
+        self.teams = []
+        self.teamsraw = {}
+        self.summary = {}
+        self.err = None
+
+    def get_minigraph(self):
+        """
+        """
+        minigraph_path = "/etc/ssw/minigraph.xml"
+
+        # Store port channel IDs	
+        for port in parse_xml(minigraph_path)['minigraph_portchannel_interfaces'].keys():
+            # Skip the 'PortChannel' prefix and store the port channel ID
+            self.teams.append(port[11:])
```

Comment:
Here I used the same logic as the previous pr to extract the team ID and only change the way to parse minigraph. I will change the logic here.